srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -30,10 +30,11 @@ global
     maxconn     {{ (150 / salt['pillar.get']('haproxy:nbproc', 2)) | round(0, 'ceil') | int }}
```

Comment:
Should this formula change?
If `nbproc` is 1, this formula doesn't help.
Consider updating it to:
```
maxconn{{ (150 / salt['pillar.get']('haproxy:nbthread', 2)) | round(0, 'ceil') | int }}
```

Reply:
Yes this should be changed, but this global maxconn value depends upon the number of S3 instances. However I am not sure why it is not reflecting here in this equation. @t7ko-seagate can you please check this once ?